testing/playwright.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from '@playwright/test';
+
+export default defineConfig({
+  testDir: './playwright/e2e',
+  timeout: 90_000,
```

Comment:
I'd like us to revisit this timeout, as I questioned it when updating cypress tests recently. I think we should figure out a more appropriate number for playwright tests. How do these tests run in terms of speed in comparison?

Also, I'm in favor of this migration, but can you explain your reasoning behind the migration? What benefits are we getting from playwright that we didn't get with cypress?